test/integration/certificates/controller_approval_test.go

```diff
@@ -49,43 +49,44 @@ func TestController_AutoApproval(t *testing.T) {
 			Organization: []string{"system:nodes"},
 		},
 	})
-	validKubeAPIServerClientKubeletUsages := []certv1beta1.KeyUsage{
-		certv1beta1.UsageDigitalSignature,
-		certv1beta1.UsageKeyEncipherment,
-		certv1beta1.UsageClientAuth,
+	validKubeAPIServerClientKubeletUsages := []certv1.KeyUsage{
+		certv1.UsageDigitalSignature,
+		certv1.UsageKeyEncipherment,
+		certv1.UsageClientAuth,
 	}
 	tests := map[string]struct {
 		signerName          string
 		request             []byte
-		usages              []certv1beta1.KeyUsage
+		usages              []certv1.KeyUsage
 		username            string
 		autoApproved        bool
 		grantNodeClient     bool
 		grantSelfNodeClient bool
 	}{
 		"should auto-approve CSR that has kube-apiserver-client-kubelet signerName and matches requirements": {
-			signerName:          certv1beta1.KubeAPIServerClientKubeletSignerName,
+			signerName:          certv1.KubeAPIServerClientKubeletSignerName,
 			request:             validKubeAPIServerClientKubeletCSR,
 			usages:              validKubeAPIServerClientKubeletUsages,
 			username:            validKubeAPIServerClientKubeletUsername,
 			grantSelfNodeClient: true,
 			autoApproved:        true,
 		},
 		"should auto-approve CSR that has kube-apiserver-client-kubelet signerName and matches requirements despite missing username if nodeclient permissions are granted": {
-			signerName:      certv1beta1.KubeAPIServerClientKubeletSignerName,
+			signerName:      certv1.KubeAPIServerClientKubeletSignerName,
 			request:         validKubeAPIServerClientKubeletCSR,
 			usages:          validKubeAPIServerClientKubeletUsages,
 			username:        "does-not-match-cn",
 			grantNodeClient: true,
 			autoApproved:    true,
 		},
-		"should not auto-approve CSR that has kube-apiserver-client-kubelet signerName that does not match requirements": {
-			signerName:   certv1beta1.KubeAPIServerClientKubeletSignerName,
-			request:      pemWithGroup("system:notnodes"),
-			autoApproved: false,
-		},
+		// usages are now required in v1.
+		//"should not auto-approve CSR that has kube-apiserver-client-kubelet signerName that does not match requirements": {
+		//	signerName:   certv1.KubeAPIServerClientKubeletSignerName,
+		//	request:      pemWithGroup("system:notnodes"),
+		//	autoApproved: false,
+		//},
```

Comment:
nit: did you forget to delete this in the second commit?

Reply:
> nit: did you forget to delete this in the second commit?

I did. now removed.